components/footer/footer.html

```diff
@@ -0,0 +1,94 @@
+<footer>
+    <div class="container-fluid">
+      <div class="row subscription">
+        <div
+          class="col-12 col-lg-6 subscription--box d-flex flex-column justify-content-center align-items-center"
+        >
+          <h2 class="d-flex flex-column subscription--box--title">
+            <span>Newsletter</span>
+            <span>Subscription</span>
+          </h2>
+          <form class="subscription--box--email">
+            <div class="input-group mb-3">
+              <input
+                type="text"
+                placeholder="shovon.khan0099@gmail.com"
+                aria-label="Recipient's username"
+                aria-describedby="basic-addon2"
+                class="subscription--box--email--input position-relative"
+              />
+              <div class="input-group-append">
+                <button
+                  type="submit"
+                  class="btn btn-dark input-group-text subscription--box--email--btn position-absolute top-0 end-0"
+                  id="basic-addon2"
+                >
+                  &#8599;
+                </button>
+              </div>
+            </div>
+          </form>
+        </div>
+        <div
+          class="col-12 col-lg-6 d-flex align-items-center justify-content-center"
+        >
+          <img
+            class="subscription--image"
+            src="./assets/images/main/american-football-player.png"
+          />
+        </div>
+      </div>
+      <div
+        class="w-100 d-flex flex-column gap-sm-0 gap-5 flex-sm-row justify-content-sm-between flex-wrap align-items-center social-bar"
```

Comment:
1- padding right and left equal0px not 10px
2-- padding top=48px and padding-bottom =82px not 42px and 42px

Reply:
@hassanosamaa done.